File: Baekjoon/구현/2012.py
# https://www.acmicpc.net/problem/2012

import sys
input = sys.stdin.readline

n = int(input())
nums = []
for _ in range(n):
    nums.append(int(sys.stdin.readline()))
nums.sort()

diff = 0
for i in range(1, n + 1):
    diff += abs(i - nums[i - 1])

print(diff)

File: Baekjoon/구현/2012.cpp
// https://www.acmicpc.net/problem/2012

#include <iostream>
#include <vector>
#include <algorithm>
using namespace std;

int main() {
    int n;
    cin >> n;

    vector<int> v(n);
    for (int i = 0; i < n; i++) {
        cin >> v[i];
    }

    sort(v.begin(), v.end());

    long long answer = 0;
    for (int i = 0; i < n; i++) {
        answer += abs(v[i] - (i + 1));
    }

    cout << answer << endl;

    return 0;
}